### PR TITLE
Add the MasterPluginEditor

### PR DESCRIPTION
**Pull request by markusguenther**
The MasterPluginEditor was missing at the moment. So this PR will add the Editor.
The PR solves one part of the issue #1262 

Related: #1262

### CODE REVIEW

File: Resources/Private/Translations/en/Main.xlf [hunk 1]
```diff
@@ -20,6 +20,12 @@
 			<trans-unit id="imageCropper__aspect-ratio-placeholder" xml:space="preserve">
 				<source>Choose an Aspect Ratio</source>
 			</trans-unit>
+			<trans-unit id="content.inspector.editors.masterPluginEditor.selectPlugin" xml:space="preserve">
+				<source>Select a Plugin</source>
```
Comment (dimaip) [inline Resources/Private/Translations/en/Main.xlf]: Just wondering, can't we use labels from the old UI here?

**Comment by dimaip** [comment]
Looks good at first glance, left a comment.

File: Resources/Private/Translations/en/Main.xlf (hunk 1 above)
Comment (markusguenther) [reply Resources/Private/Translations/en/Main.xlf]: Sure but yesterday someone mentioned that the dependencies should be low so I thought it is better to have the labels. But can change that.

**Comment by markusguenther** [comment]
Yeah still struggling with the data fetching... the dimensions should be in a different format on PHP side.

**Pull request merged**